### DISCUSSION TITLE
Is it possible to target pseudo-elements in attributify mode?

### DISCUSSION THREAD

**Discussion by vndre** [windicss/windicss#353] [Q&A]
I have tried this attributes with no luck:

```
w:after="content-['yes'] block bg-red-500"
className="before:content-['yes'] before:block before:bg-red-500"
```

**Comment by shiftgeist** [comment]
I don't think this is possible since it only reduces the class groups (`bg-green-300 bg-opacity-60` -> `bg="green-300 opacity-60"`) on the current object. The documentation does not appear to give any indication at targeting other elements.

**Reply by vndre** [reply]
I can see there is a "after" variant, and attributify docs says the syntax is: `(variant[:-]{1})*key?` so `after=""` should already work?

**Reply by shiftgeist** [reply]
I get your point now.

[attributify](https://windicss.org/posts/attributify.html) might be limited to the ones that are present on the page?

Reference for `after=""` appears on https://windicss.org/utilities/variants.html#pseudo-elements (for anyone looking into this).

`group-hover` support was also added afterwards: https://github.com/windicss/windicss/issues/308

**Reply by vndre** [reply]
My guess is supported attributes in this mode have to be added manually like they did with `group-hover`. Shall I open an issue for this?

**Reply by vndre** [reply]
Tailwind's JIT recently added support for pseudo-elements: https://github.com/tailwindlabs/tailwindcss/pull/4461

**Reply by shiftgeist** [reply]
At the very least `pseudo-elements` (and `group-hover`) are missing from the attributify docs.
